pkg/storage/azurecdn/storage.go

```diff
@@ -0,0 +1,82 @@
+package azurecdn
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/url"
+	"strings"
+
+	"github.com/Azure/azure-storage-blob-go/2017-07-29/azblob"
+)
+
+// Storage implements (github.com/gomods/athens/pkg/storage).Saver and
+// also provides a function to fetch the location of a module
+type Storage struct {
+	accountURL *url.URL
+	cred       azblob.Credential
+}
+
+// New creates a new azure CDN saver
+func New(accountName, accountKey string) (*Storage, error) {
+	u, err := url.Parse(fmt.Sprintf("https://%s.blob.core.windows.net", accountName))
```

Comment:
is accountName URI safe? does it need sanitizing?

Reply:
we'll decide it when we create the account, so we'll need to make it URI safe when we do